src/main/java/org/folio/orders/rest/exceptions/CustomHttpException.java

```diff
@@ -0,0 +1,23 @@
+package org.folio.orders.rest.exceptions;
+
+import org.folio.orders.utils.ErrorCodes;
+
+public class CustomHttpException extends Exception {
```

Comment:
It may be easier to expand HttpException to accept an ErrorCode than create a whole new class, but it's up to you.  

Other things to consider if you decide to keep this class:
* if you're creating a specialized variant of HttpException, why not extend HttpException?
* it's generally good practice to call the super constructor
* it would be nice to provide a toString() implementation